site/app/templates/polls/ViewPoll.twig

```diff
@@ -97,6 +107,34 @@
                     <button type="submit" class="btn btn-primary student-submit">Submit</button>
                 {% endif %}
             </form>
+            {% if not user_admin and poll.isOpen() and poll.getAllowsCustomResponses() %}
+                <div class = 'info-wrapper custom-response-wrapper'>
+                    <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
+                    <input type="hidden" name="poll_id" value="{{ poll.getId() }}"/>
+                    <br />
+                    <div class="response-container">
+                        <textarea
+                            aria-label="Response text"
+                            class="poll-response custom-poll-response"
+                            name="custom-response"
+                            placeholder="Enter response here..."
+                            rows="10" cols="30"></textarea>
+                    </div>
+                    <i>
+                        Are you certain you want to add a response? It will be visible to every student.<br />
```

Comment:
Instead of asking a question as if this is a confirmation prompt, I think this should just be a line of informational text.
```suggestion
                        This response will be visible to every student.<br />
```